### PR TITLE
ZBUG-995 : Attachment with content-type with text/plain and text/html files if have content-disposition: inline will be shown has normal attachment in legacy for downloading or viewing

### PR DESCRIPTION
**Pull request by auminpatel**

### CODE REVIEW

File: WebRoot/js/zimbraMail/mail/model/ZmMailMsg.js [hunk 1]
```diff
@@ -2152,14 +2152,13 @@ function(findHits, includeInlineImages, includeInlineAtts) {
 
 			if (!this.isRealAttachment(attach) ||
 					(attach.contentType.match(/^image/) && attach.contentId && attach.foundInMsgBody && !includeInlineImages) ||
-					(attach.contentDisposition == "inline" && attach.fileName && ZmMimeTable.isRenderable(attach.contentType, true) && !includeInlineAtts) ||
+					(attach.contentDisposition == "inline" && attach.fileName && ZmMimeTable.textIsRenderable(attach.contentType, attach.isBody) && !includeInlineAtts) ||
```
Comment (BhavinBathani) [inline WebRoot/js/zimbraMail/mail/model/ZmMailMsg.js]: function name should be like isTextRenderable or isRenderableText ?

Comment (auminpatel) [reply WebRoot/js/zimbraMail/mail/model/ZmMailMsg.js]: Done

File: WebRoot/js/zimbraMail/mail/model/ZmMailMsg.js [hunk 2]
```diff
@@ -2152,14 +2152,13 @@ function(findHits, includeInlineImages, includeInlineAtts) {
 
 			if (!this.isRealAttachment(attach) ||
 					(attach.contentType.match(/^image/) && attach.contentId && attach.foundInMsgBody && !includeInlineImages) ||
-					(attach.contentDisposition == "inline" && attach.fileName && ZmMimeTable.isRenderable(attach.contentType, true) && !includeInlineAtts) ||
+					(attach.contentDisposition == "inline" && attach.fileName && ZmMimeTable.isRenderableText(attach.contentType, attach.isBody) && !includeInlineAtts) ||
 					(attach.contentDisposition == "inline" && attach.contentType === "application/pdf" && attach.contentId && attach.foundInMsgBody)) {
```
Comment (silentsakky) [inline WebRoot/js/zimbraMail/mail/model/ZmMailMsg.js]: do we still need this line of code?

Comment (auminpatel) [reply WebRoot/js/zimbraMail/mail/model/ZmMailMsg.js]: for pdf we need this

**Pull request merged**